src/api/app/models/event/service_fail.rb

```diff
@@ -21,6 +21,40 @@ def set_payload(attribs, keys)
       attribs['error'] = attribs['error'][0..800]
       super(attribs, keys)
     end
+
+    def metric_measurement
+      'service'
+    end
+
+    def metric_tags
+      error = case payload['error']
+              when start_with?('bad link:')
+                'bad_link'
+              when /400 remote error:.*.service  No such file or directory/
+                'service_missing'
+              when start_with?('404 remote error:'), start_with?('FileNotFoundError:'), /ERROR 404: Not Found/, /No such file or directory/
+                'not_found'
+              when start_with?(' 400 remote error:')
+                'remote_error_400'
+              when /ERROR 403: Forbidden/
```

Comment:
I'm not sure this will be the same forbidden every time we see this. I would count this as `unknown`